wagtail/admin/templates/wagtailadmin/skeleton.html

```diff
@@ -9,7 +9,7 @@
         <meta name="description" content="" />
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
         <meta name="robots" content="noindex" />
-
+        <meta name="color-scheme" content="{% admin_theme_user %}">
```

Comment:
Could you add a test please, e.g. in [`wagtail.admin.tests.test_templatetags`](https://github.com/wagtail/wagtail/blob/main/wagtail/admin/tests/test_templatetags.py)